### PR TITLE
Added a Tags renderer at the end of the blog posts

### PR DESCRIPTION
**Pull request by mihirgupta0900**
## I ensure I have done the following steps

- [x] Star the repository :star:
- [x] Followed the [author](https://github.com/piyush97):white_check_mark:
- [x] Tested the code properly :white_check_mark:
- [x] Added the required feature :white_check_mark:

### CODE REVIEW

File: src/components/Tags.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styled from '@xstyled/styled-components';
+
+const InlineH3 = styled.span`
+  font-size: 1.17em;
+`;
```
Comment (piyush97) [inline src/components/Tags.js:6]: Need to make Tag component with Styling is required, the current change only shows text, not Tags with text (Components):
image

Styling the component properly would be appreciated :)

**Review by piyush97 — CHANGES_REQUESTED** [review]
Need to make Tag component with Styling is required, the current change only shows text, not Tags with text (Components):
image

Styling the component properly would be appreciated :)

File: src/components/Tags.js (hunk 1 above)
Comment (mihirgupta0900) [reply src/components/Tags.js:6]: Sure! Do you have any specific design requests? Or should I use what's in the image?

Comment (piyush97) [reply src/components/Tags.js:6]: @mihirgupta0900 you can use your own creativity and create a design which looks good! 
Make sure you follow the color scheme! :) 
All the best!

**Review by piyush97 — APPROVED** [review]
Good work! @mihirgupta0900 🎉

**Comment by piyush97** [comment]
@all-contributors please add @mihirgupta0900 as designer,code

**Comment by piyush97** [comment]
@all-contributors please add @mihirgupta0900 for design and code

**Pull request merged**